app/src/utils/apiHelpers.test.tsx

```diff
@@ -0,0 +1,212 @@
+  });
+});
+
+describe("getNumDigitizedItems", () => {
+  it("returns the correct numDigitizedItems", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve({
+        count: {
+          $: 78,
+        },
+      })
+    );
+    const result = await getNumDigitizedItems();
+
+    expect(result).toEqual("78");
+  });
+
+  it("returns the fallback numDigitizedItems on empty response", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(Promise.resolve({}));
+
+    const result = await getNumDigitizedItems();
+
+    expect(result).toEqual(
+      defaultFeaturedItem["development"].numberOfDigitizedItems
+    );
+  });
+});
+
+describe("getItemsCountFromUUIDs", () => {
+  it("should return the correct numItems for each UUID", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve({
+        nyplAPI: {
+          response: {
+            counts: {
+              count: [
+                { uuid: { $: "uuid1" }, count_value: { $: "10" } },
+                { uuid: { $: "uuid2" }, count_value: { $: "45" } },
+              ],
+            },
+          },
+        },
+      })
+    );
+
+    const uuids = ["uuid1", "uuid2"];
+
+    const results = await getItemsCountFromUUIDs(uuids);
+
+    expect(results).toEqual({
+      uuid1: "10",
+      uuid2: "45",
+    });
+  });
+
+  it("should handle missing count field gracefully", async () => {
+    const uuids = ["uuid1", "uuid2", "uuid3"];
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve({
+        nyplAPI: {
+          response: {
+            counts: {
+              count: [
+                { uuid: { $: "uuid1" }, count_value: { $: "10" } },
+                { uuid: { $: "uuid3" }, count_value: { $: "60" } },
+              ],
+            },
+          },
+        },
+      })
+    );
+    const results = await getItemsCountFromUUIDs(uuids);
+    expect(results).toEqual({
+      uuid1: "10",
+      uuid3: "60",
+    });
+  });
+});
+
+describe("getRandomFeaturedItem", () => {
+  it("returns expected item", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve(mockFeaturedItemResponse)
+    );
+    const item = await getRandomFeaturedItem();
+    expect(item).toEqual(mockFeaturedItemResponse);
+    expect(item).toHaveProperty("capture");
+  });
+});
+
+describe("getFeaturedImage", () => {
+  it("returns expected item", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve(defaultFeaturedItem["production"].featuredItem)
+    );
+    const item = await getFeaturedImage();
+    expect(item).toHaveProperty("imageID");
+    expect(item.imageID).toEqual("482815");
+  });
+});
+
+describe("getItemData", () => {
+  it("returns expected item", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve(mockItemResponse)
+    );
+    const item = await getRandomFeaturedItem();
```

Comment:
This should be `getItemData`. This brings up a good point in these tests because this test still passes: many of these functions are wrappers around `apiResponse`. Essentially, all these tests are mocking and testing that internal API call and its response. Like other tests above, add an expectation that a specific URL was called. This doesn't entirely address my concern but gets closer to really testing the functions.